# Mission Log Message
std_msgs/Header header

# Mission ID
string mission_id

# Log message
string message

# Cryptographic hash of the previous log
string previous_hash
